packages/palette/src/elements/Cards/SmallCard.tsx

```diff
@@ -1,11 +1,83 @@
 import React from "react"
+import { space } from "../../helpers"
+import { Box } from "../Box"
+import { Flex } from "../Flex"
+import { Image } from "../Image"
+import { Spacer } from "../Spacer"
+import { Sans } from "../Typography"
+import { CardTag } from "./CardTag"
 import { SmallCardProps } from "./SmallCard.shared"
 
 /**
  * `SmallCard` is a card with a layout one square image on the left,
  * one tall or two square images on the right, and text for title and subtitle
  * at the bottom.
  */
-export const SmallCard: React.FC<SmallCardProps> = () => {
-  return null
+export const SmallCard: React.FC<SmallCardProps> = ({
+  images,
+  title,
+  subtitle,
+  tag,
+}) => {
+  return (
+    <Box width="100%" position="relative">
+      <Box width="100%" height="0" pt="67%" position="relative">
+        <Flex
+          flexDirection="row"
+          width="100%"
+          height="100%"
+          position="absolute"
+          top="0"
+          borderRadius="4px"
+          overflow="hidden"
+        >
+          <Box backgroundColor="black10" width="67%" overflow="hidden" mr="1px">
```

Comment:
Copied this layout idea from iOs version. The case that the VR has no valid images does not seem to be handled. So not sure what should happen there. But I guess the header image is required from CMS for publishing VRs. So hopefully that will not be the case often if ever.